lib/AST/ASTMangler.cpp

```diff
@@ -450,6 +450,9 @@ std::string ASTMangler::mangleReabstractionThunkHelper(
   assert(ThunkType->getPatternSubstitutions().empty() && "not implemented");
   GenericSignature GenSig = ThunkType->getInvocationGenericSignature();
 
+  // Reabstraction thunks never reference inverse conformances.
```

Comment:
Why not? It’s possible to end up with two thunks that only differ in Copyable requirements and this will collapse them down to one symbol no?

Reply:
Also reabstraction thunk mangling is not ABI…

Reply:
Hmm. Copyable shouldn't have any impact on the actual reabstraction thunk, no? But perhaps we do have to keep these separate...